src/nfa/operations.cc

```diff
@@ -772,17 +758,17 @@ Nfa Mata::Nfa::reduce(const Nfa &aut, bool trim_input, StateToStateMap *state_ma
                                  " received an unknown value of the \"algorithm\" key: " + algorithm);
     }
 
-    if (state_map) {
-        state_map->clear();
+    if (state_renaming) {
+        state_renaming->clear();
         if (trim_input) {
-            for (const auto& trimmed_mapping : trimmed_state_map) {
+            for (const auto& trimmed_mapping : trimmed_state_renaming) {
```

Comment:
just state_renanaming?

Reply:
The option will be removed in a future PR.

Reply:
okok